bin/npm-ci-publish.js

```diff
@@ -10,6 +10,15 @@ import {
 import { writeFileSync, unlinkSync } from 'fs';
 import { execSync } from 'child_process';
 
+const program = require('commander');
+
+// eslint-disable-next-line
+program.version(require('../../package').version)
```

Comment:
is it two levels up, not one?

Reply:
I know, it's weird. I copied from the existing commands, and that's how it was. I'm not sure how it actually works, I assumed it was some `commander` magic.